bugbug/utils.py

```diff
@@ -513,3 +513,14 @@ def escape_markdown(text: str) -> str:
         .replace(")", "\\)")
         .replace("|", "\\|")
     )
+
+
+# Remove business days from date range
```

Comment:
You could convert this to docstrings (please consider using Google Style Python Docstrings).